src/App.js

```diff
@@ -7,13 +7,15 @@ import Start from './components/Start';
 import Info from './components/Info';
 
 import startImage from './images/start_screen.png';
+import { clearChoicesRecord } from "./utils/utils";
 
 function App() {
   const [isInStart, setIsInStart] = useState(true);
   const [questionsSeen, setQuestionsSeen] = useState(-1);
   const [isInInfo, setIsInInfo] = useState(false);
 
   const startGame = () => {
+    clearChoicesRecord()
```

Comment:
nice!

Reply:
we'll probably want a separate ticket for badges won to be stored in local storage and that wouldn't be cleared on game start

Reply:
Good idea, I can circle back to that